rasa/core/policies/policy.py

```diff
@@ -420,6 +424,8 @@ def __init__(
                 you return as they can potentially influence the conversation flow.
             is_end_to_end_prediction: `True` if the prediction used the text of the
                 user message instead of the intent.
+            is_no_user_prediction: `True` if the prediction uses neither the text
+                of the user message nor the intent.
```

Comment:
let's please add something along these lines for clarification. I don't think it's clear for everyone otherwise.
```suggestion
                of the user message nor the intent. This is for the example the case for happy loop paths.
```